packages/data/CHANGELOG.md

```diff
@@ -2,69 +2,76 @@
 
 ## Unreleased
 
+### New Features
+
+-   Added new `register` factory function for registering a standard `@wordpress/data` store definition ([#26655](https://github.com/WordPress/gutenberg/pull/26655)).
```

Comment:
not sure why you called this a "factory function"?

Reply:
Copy and paste? 😃